bigquery/tests/unit/test_table.py

```diff
@@ -1561,14 +1561,14 @@ def _row_data(row):
         self.assertEqual(req['path'], '/%s' % PATH)
         self.assertEqual(req['data'], SENT)
 
-    @mock.patch('google.auth.transport.requests.AuthorizedSession')
-    def test__make_transport(self, session_factory):
-        client = mock.Mock(spec=[u'_credentials'])
+    def test__get_transport(self):
+        client = mock.Mock(spec=[u'_credentials', '_http'])
+        client._http = mock.sentinel.http
```

Comment:
This is lame and nitpicky, and you don't need to change anything, but...do you prefer two statements over:

```python
client = mock.Mock(
    _http=mock.sentinel.http, spec=[u'_credentials', '_http'])
```